packages/react/src/components/Menu/MenuDivider.tsx

```diff
@@ -9,16 +9,25 @@ import {
   UIComponentProps,
   ColorComponentProps,
   commonPropTypes,
+  customPropTypes,
+  childrenExist,
+  ChildrenComponentProps,
+  ContentComponentProps,
 } from '../../lib'
-import { ReactProps } from '../../types'
+import { ReactProps, ShorthandValue } from '../../types'
 
-export interface MenuDividerProps extends UIComponentProps, ColorComponentProps {
+export interface MenuDividerProps
+  extends UIComponentProps,
+    ChildrenComponentProps,
+    ContentComponentProps,
+    ColorComponentProps {
   /**
    * Accessibility behavior if overridden by the user.
    * @default menuDividerBehavior
    */
   accessibility?: Accessibility
 
+  icon?: ShorthandValue
```

Comment:
I don't see how we are handling the `icon` prop.. Are we adding both content and icon, or the intention is just the `content` prop to be added?

Reply:
Cruft, will remove it 👍